FreshAir/UI/RZFReleaseNoteCollectionViewCell.m

```diff
@@ -50,28 +69,52 @@ - (instancetype)initWithFrame:(CGRect)frame
     
     if ( self ) {
         self.backgroundColor = [UIColor clearColor];
-        
-        [self setupScreenshotImageView];
-        [self setupInfoContainerView];
     }
     
     return self;
 }
 
 # pragma mark - Setup
 
+- (void)prepareForReuse {
```

Comment:
I'd call `[super prepareForReuse]` as well. Not sure it matters in this case, but UIKit likes super to be called unless the documentation states otherwise.